app/kafka/manager/actor/cluster/KafkaStateActor.scala

```diff
@@ -284,7 +625,8 @@ case class OffsetCacheActive(curator: CuratorFramework,
 
   protected def getTopicDescription(topic: String, interactive: Boolean) : Option[TopicDescription] = topicDescriptions(topic, interactive)
   
-  def start():  Unit = {
+  override def start():  Unit = {
+    super.start()
```

Comment:
True, but kafka managed offsets cache isn't polling, it is just consuming from a topic in Kafka.  Should we introduce another feature to turn it on / off?